src/plugins/kibana/server/lib/ingest_build_request.js

```diff
@@ -0,0 +1,24 @@
+const _ = require('lodash');
```

Comment:
Tests for this lib would also be helpful for understanding the expected input/output here. For instance I don't know why the pipeline has a `rootObject` property, or why there's only a single expected doc?